From de710c2d74ab1b32f3e976e58d6cb3ee248f73af Mon Sep 17 00:00:00 2001
From: "David E. Box" <david.e.box@linux.intel.com>
Date: Fri, 28 Feb 2014 18:40:51 -0800
Subject: [PATCH 3/5] x86, iosf: Add dummy functions for loadable modules

Some loadable modules only need IOSF access on the platforms where it exists.
Provide dummy functions to allow these modules to compile and load on the
platforms where it doesn't exist.

Change-Id: I4c1672d21487762bb8e32ebf20c04b61d5acb52f
Signed-off-by: David E. Box <david.e.box@linux.intel.com>
Link: http://lkml.kernel.org/r/1393641652-7222-2-git-send-email-david.e.box@linux.intel.com
Signed-off-by: H. Peter Anvin <hpa@linux.intel.com>
---
 arch/x86/include/asm/iosf_mbi.h |    1 -
 1 file changed, 1 deletion(-)

--- linux.orig/arch/x86/include/asm/iosf_mbi.h
+++ linux/arch/x86/include/asm/iosf_mbi.h
@@ -73,7 +73,6 @@
 #define QRK_MBI_SOC_WRITE	0x07
 
 #if IS_ENABLED(CONFIG_IOSF_MBI)
-
 bool iosf_mbi_available(void);
 
 /**
